public/typo3conf/LocalConfiguration.php

```diff
@@ -0,0 +1,177 @@
+<?php
```

Comment:
Unfortunately, the LocalConfiguration.php cannot be part of an extension.

The extension configuration can only be written to some INSTALL.md or another kind of documentation.

Reply:
Ok, I have placed it in README.md and move this file as the template of the LocalConfig.php, which can be used as a reference during installation.